ledger/db/src/tx_out_store.rs

```diff
@@ -790,7 +790,7 @@ pub mod tx_out_store_tests {
         membership_proofs::{hash_leaf, hash_nodes, Range, NIL_HASH},
         onetime_keys::*,
         tx::TxOut,
-        Amount,
+        Amount, MemoPayload,
```

Comment:
i ran rustfmt in the container, and it didn't change this

i think this conforms to rustfmt rules because all the other items are imports from submodules, and Amount and MemoPayload are not submodules